packages/wallet-sdk/src/core/communicator/Communicator.ts

```diff
@@ -96,5 +110,40 @@ export class Communicator {
         if (!this.popup) throw standardErrors.rpc.internal();
         return this.popup;
       });
-  }
+  };
+
+  /**
```

Comment:
there are some lines that are quite subtle.
wish there's cleaner solution for this 😞

Reply:
I'd like to better understand this comment